code/datums/hud/critter.dm

```diff
@@ -375,30 +375,30 @@
 
 /// updates bleeding hud element
 /datum/hud/critter/proc/update_blood_indicator()
-	if (!src.bleeding) return //doesn't have a hud element to update
-	if (isdead(src.master))
-		src.bleeding.icon_state = "blood0"
-		src.bleeding.tooltipTheme = "healthDam healthDam0"
-		return
```

Comment:
you are removing  runtime hotfix, this will cause breakage

Reply:
should be mediated now, yell at me in discord if it's not